src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4861blockcommentstyle/InputCommentsIndentationInSwitchBlock.java

```diff
@@ -18,75 +19,73 @@ private static void fooSwitch() {
       case "3":
       /* // odd indentation comment
          *  */
-        // violation 2 lines above '.* indentation should be the same level as line 22.'
+        // violation 2 lines above '.* indentation should be the same level as line 23.'
         foo1();
         /* com */
         break;
       case "5":
-        // violation 2 lines below '.* indentation should be the same level as line 27, 29.'
+        // violation 2 lines below '.* indentation should be the same level as line 30.'
         foo1();
            // odd indentation comment
+        break;
       case "6":
         int k = 7;
-        // fall through
+        break;
```

Comment:
Why did you replace `// fall through` with `break` here, but didn't do it few lines below (line 43)?

Reply:
sorry my bad. In some places it was giving `fall through` violation so I had to add break statement everywhere, I might have accidentally removed the comment here and added the break statement here. I have revert it, the comment is back now.

It is strange that if we have `// fall through` comment instead of `break;` then we don't get the fall through violation.

In some places this comment was inside the block so I have moved them outside to prevent violation. Example: https://github.com/checkstyle/checkstyle/pull/15398/files?diff=unified&w=0#diff-a2fac517f687a7b67e6183b98412714725d08a574219ae15e00d974d9ad23532R44-R47